ams/api/src/main/java/com/netease/arctic/ams/api/plugin/PluginManager.java

```diff
@@ -0,0 +1,16 @@
+package com.netease.arctic.ams.api.plugin;
+
+import java.util.List;
+
+public interface PluginManager<T extends Plugin> {
```

Comment:
All interfaces and their methods require comments.

Reply:
done